src/Daemon/SentryWriter.h

```diff
@@ -14,14 +14,32 @@ class StackTrace;
 ///
 /// It is possible to send those reports to your own sentry account or account of consulting company you hired
 /// by overriding "send_crash_reports.endpoint" setting. "send_crash_reports.debug" setting will allow to do that for
-namespace SentryWriter
+class SentryWriter
```

Comment:
As I am trying to be cautious about what data I share with companies, I think it is a must to mention that this class from now on will also send some exceptions too. Could you please extend its documentation briefly?

Reply:
Documentation for the class you mean?

Documentation itself is already updated:

```
`send_logical_errors` – `LOGICAL_ERROR` is like an `assert`, it is a bug in ClickHouse. This boolean flag enables sending this exceptions to sentry (default: `false`).
```

Reply:
Updated brief comment for class as well.

Reply:
> Documentation for the class you mean?

Yes, I meant that, thanks for updating it.